scripts/requirements-doc.txt

```diff
@@ -2,7 +2,7 @@
 
 breathe>=4.9.1,<4.15.0
 docutils>=0.14
-sphinx>=1.7.5,<3.0
+sphinx>=1.7.5, != 2.4.0, <3.0
```

Comment:
@carlescufi  is `!= 2.4.0` still reqired?

Reply:
No, this should work:

```
breathe>=4.9.1,<4.15.0
docutils>=0.14
sphinx>=1.7.5,<3.0
```